metricbeat/module/ceph/cluster_status/cluster_status_integration_test.go

```diff
@@ -22,13 +22,22 @@ import (
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
 func TestData(t *testing.T) {
-	f := mbtest.NewEventsFetcher(t, getConfig())
-	err := mbtest.WriteEvents(f, t)
-	if err != nil {
+	t.Skip("Skipping test. After migration to ReportV2 call to mbtest.ReportingFetchV2(f) fails (which replaced 'f := mbtest.NewEventsFetcher(t, getConfig())' call).")
```

Comment:
`TestData` needed to be skip. After migrating this code to use `ReporterV2` interface, it seems that something is not working the same way and causes the test to fail 100% times in line #36 https://github.com/elastic/beats/pull/10993/files#diff-2b632f9f0d99e92969b3e092937d03dcR36

Reply:
Umm, what I am going to say can affect all these PRs :grimacing: but in modules migrated previously `TestData` was basically only these lines:
```
        config := getConfig()
        f := mbtest.NewReportingMetricSetV2(t, config)
        err := mbtest.WriteEventsReporterV2(f, t, ".")
        if err != nil {
                t.Fatal("write", err)
        }
```

The advantage is that nothing is really done (apart of instantiating the metricset) if no `-data` flag is provided. Also `WriteEventsReporterV2` already does the check for errors and non empty events, so the lines failing here wouldn't be necessary.

I'd say to go back to this version without calling `ReportingFetchV2` here.

Reply:
Good catch. @sayden Is there a reason we need the additional lines?

Reply:
I thought they were mandatory. This entire block was copy pasted from other modules that are already merged (outside of the scope of the migration). So my guess now is that they aren't necessary at all

Reply:
I don't mind about changing all opened PR's anyways. It just that it was written like this even in the "tutorial" that the community is using to help in the migration